packages/core/src/modules/credentials/services/CredentialService.ts

```diff
@@ -672,6 +672,15 @@ export class CredentialService {
       credentialDefinition,
       revocationRegistryDefinition,
     })
+
+    // If we have the rev_reg_id, then we also want to set the cred_rev_id, which only can be fetched by retrieving credential from indy
+    if (indyCredential.rev_reg_id) {
+      const credential = await this.indyHolderService.getCredential(credentialId)
+
+      credentialRecord.credentialRevocationId = credential.cred_rev_id?.toString()
```

Comment:
Is cred_rev_id not tupe string already?

Reply:
According to the [@types/indy-sdk](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/74b19cf32a1b2f10958f5729f798245afb1125f5/types/indy-sdk/index.d.ts#L455) it is a number

Reply:
We probably need to test this--as the type definition may be incorrect. I know for sure that the revocation registry delta contains the cred_rev_id's in the revoked field are numbers, but I _think_ the cred_rev_id is a string when doing getCredential. 
@TheTreek if you could that'd be great :)

Reply:
Testing confirmed that it is a string

Reply:
Created pr for it here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/59079